python/tvm/relay/frontend/pytorch.py

```diff
@@ -3254,10 +3259,14 @@ def from_pytorch(script_module, input_shapes, custom_convert_map=None, default_d
         TorchScripted PyTorch graph
         Note: We currently only support traces (ie: torch.jit.trace(model, input))
 
-    input_shapes : List of tuples of input name and input dimensions
-        Graph level input shape list
+    input_infos: List of tuples of input name and (input dimensions, input types)
+        Graph level input shape and type list
```

Comment:
Can you also add that `(name, shape)` is also allowed? Since this is the more common use case.